automatic/webstorm/update.ps1

```diff
@@ -0,0 +1,38 @@
+﻿Import-Module AU
+
+function global:au_GetLatest {
+    $productName = 'WebStorm'
+    $releaseUrl = 'https://www.jetbrains.com/updates/updates.xml'
+    $downloadUrl = 'https://download.jetbrains.com/webstorm/WebStorm-$($version).exe'
+
+    [xml] $updates = (New-Object System.Net.WebClient).DownloadString($releaseUrl)
+    $versionInfo = $updates.products.product `
+        | Where-Object { $_.name -eq $productName } `
+        | ForEach-Object { $_.channel } | Where-Object { $_.id -eq 'WS_Release' } `
+        | ForEach-Object { $_.build } `
+        | Sort-Object { [version] $_.number } `
+        | Where-Object { $_.version -notmatch 'EAP' } `
+        | Select-Object -Last 1
```

Comment:
Is it? I am using the same logic in the DataGrip package. If you have a better way of getting to the same information, I would change it.